lizmap/modules/lizmap/classes/lizmapRepository.class.php

```diff
@@ -163,4 +202,52 @@ public function getProjects()
 
         return $projects;
     }
+
+    /**
+     * Get the repository projects metadata.
+     *
+     * @return ProjectMetadata[]
+     */
+    public function getProjectsMetadata()
+    {
+        $data = array();
+        $dir = $this->getPath();
+
+        if (is_dir($dir)) {
+            if ($dh = opendir($dir)) {
+                $cfgFiles = array();
+                $qgsFiles = array();
+                while (($file = readdir($dh)) !== false) {
+                    if (substr($file, -3) == 'cfg') {
+                        $cfgFiles[] = $file;
+                    }
+                    if (substr($file, -3) == 'qgs') {
+                        $qgsFiles[] = $file;
+                    }
+                }
+                closedir($dh);
+
+                foreach ($qgsFiles as $qgsFile) {
+                    $proj = null;
+                    if (in_array($qgsFile.'.cfg', $cfgFiles)) {
+                        try {
+                            // Get project
+                            $keepReference = false;
+                            $proj = $this->getProject(substr($qgsFile, 0, -4), $keepReference);
```

Comment:
if you modify https://github.com/drillinP/lizmap-web-client/blob/backport-2199-to-release_3_3/lizmap/modules/lizmap/classes/lizmap.class.php#L280
be sure to use 
`lizmap::getProject(substr($qgsFile, 0, -4), $keepReference)` instead